manifests/c/ctrl-f/userdiag/0.1.1/ctrl-f.userdiag.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ctrl-f.userdiag
+PackageVersion: 0.1.1
+InstallerSwitches:
+  Silent: /s
+Installers:
+- Architecture: neutral
+  InstallerType: exe
+  InstallerUrl: https://userdiag.com/download?winGet
+  InstallerSha256: A24E199305150F3CC981F5E23FFAE8840C21A9105D956CBF61B1AE6A50A2BBB8

```

```suggestion
  InstallerSha256: A24E199305150F3CC981F5E23FFAE8840C21A9105D956CBF61B1AE6A50A2BBB8
  Scope: user
```
Application is installed to `C:\Users\%USERNAME%\AppData\Roaming\UserDiag`